config/inputs/io_nvme_input.txt

```diff
@@ -1,4 +1,5 @@
 [io_nvme]
+device = "nvme0"
```

Comment:
can this be added in first commit itself

Reply:
Done all the changes are now in one commit